plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPlugin.java

```diff
@@ -33,6 +34,6 @@ public Iterable<ConnectorFactory> getConnectorFactories()
     @Override
     public Set<Class<?>> getFunctions()
     {
-        return ImmutableSet.of(IcebergThetaSketchForStats.class);
+        return ImmutableSet.of(IcebergThetaSketchForStats.class, IcebergBucketFunction.class);
```

Comment:
This should be exposed as a catalog function, not a global function.

Reply:
@martint thank you for reviewing! I'm not sure how to do this however? Is there an example of a function exposed as a catalog function in another connector I could look at to understand how to do this?

Reply:
I think you can check `random_string` function in Faker.